spec/ruby/core/process/initialize_spec.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe 'Process::Status#initialize' do
+  it 'initializes a new Process::Status instance' do
+    Process::Status.new.should be_an_instance_of(Process::Status)
+  end
+
+  it 'initializes Process::Status with a PID' do
```

Comment:
There's not a lot that would be simpler with shared specs. This type of spec (ie initialize, check initialized) is difficult to simplify. Your approach in the current version of the specs here is fine.

The purpose of the specs is to be [correct, clear, consistent, and simple](https://groups.google.com/d/msg/rubyspec/NqCC9NtfFVs/YvcqGQ54bC0J), in that order. A more verbose spec can be all of those at once.